wger/exercises/tests/api/test_exercise_info_search_api.py

```diff
@@ -0,0 +1,129 @@
+# This file is part of wger Workout Manager.
+#
+# wger Workout Manager is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# wger Workout Manager is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Workout Manager.  If not, see <http://www.gnu.org/licenses/>.
+
+# Third Party
+from rest_framework import status
+
+# wger
+from wger.core.tests.api_base_test import ApiBaseTestCase
+from wger.core.tests.base_testcase import BaseTestCase
+
+
+class ExerciseInfoFilterApiTestCase(BaseTestCase, ApiBaseTestCase):
+    url = '/api/v2/exerciseinfo/'
+
+    def setUp(self):
+        super().setUp()
+        self.init_media_root()
+
+    def _results(self, response):
+        if isinstance(response.data, dict) and 'results' in response.data:
+            return response.data['results']
+        return response.data
+
+    def _has_translation_name(self, item, expected_name: str) -> bool:
+        for t in item.get('translations', []):
+            if t.get('name') == expected_name:
+                return True
+        return False
+
+    def test_basic_search_logged_out(self):
+        """
+        Logged-out users can search via name__search and language__code
+        """
+        response = self.client.get(self.url + '?name__search=exercise&language__code=en')
```

Comment:
the smallest of nitpicks, you can also write it like this

```python
response = self.client.get(
    reverse('exerciseinfo-list'),
    {'name__search': 'foo', 'language__code': 'bar'},
)
```

I find it a bit cleaner